lib/commands/cloud-projects.ts

```diff
@@ -27,31 +27,40 @@ export class CloudListProjectsCommand implements ICommand {
 	constructor(private $logger: ILogger,
 		private $remoteProjectService: IRemoteProjectService,
 		private $prompter: IPrompter,
+		private $options: IOptions,
 		private $errors: IErrors) { }
 
 	allowedParameters: ICommandParameter[] = [new SolutionIdCommandParameter(this.$remoteProjectService)];
 
-	private printList(solutionName: string, list: Server.IWorkspaceItemData[]): void {
-		let headers =  ["#", "Project Name"];
-		let data = list.map((proj: Server.IWorkspaceItemData, index: number) => [(++index).toString(), proj.Name]);
+	private printList(names: string[], config: {isProject: boolean, solutionName?: string}): void {
+		let isProject = config.isProject;
+		let headers =  ["#", `${isProject ? 'Project' : 'Solution'} name`];
+		let data = names.map((name: string, index: number) => [(++index).toString(), name]);
 		let table = commonHelpers.createTable(headers, data);
-		this.$logger.out(`Projects for ${solutionName} solution:`);
+		if (isProject) {
+			this.$logger.out(`Projects for ${config.solutionName} solution:`);
```

Comment:
it is possible to have `config.isProject` set to true without `config.solutionName` - in this case you will print `Projects for undefined solution`
Is this expected?